art/art.py

```diff
@@ -487,6 +490,8 @@ def text2art(text, font=DEFAULT_FONT, chr_ignore=True, decoration=None):
     font = font.lower()
     if font != "mix":
         font = indirect_font(font, text)
+        if show_font:
+            print("#font name : %s\n" % font)
```

Comment:
use `.format` instead of this method ;-)

Reply:
🆗, I got it.